docs/source/user-guide/expressions.md

```diff
@@ -179,26 +179,28 @@ Unlike to some databases the math functions in Datafusion works the same way as
+| Function                             | Notes                                                                                                                                              |
+| ------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| array_append(array, element)         | Appends an element to the end of an array. `array_append([1, 2, 3], 4) -> [1, 2, 3, 4]`                                                            |
+| array_concat(array[, ..., array_n])  | Concatenates arrays. `array_concat([1, 2, 3], [4, 5, 6]) -> [1, 2, 3, 4, 5, 6]`                                                                    |
+| array_has(array, element)            | Returns true if the array contains the element `array_has([1,2,3], 1) -> true`                                                                     |
+| array_has_all(array, sub-array)      | Returns true if all elements of sub-array exist in array `array_has_all([1,2,3], [1,3]) -> true`                                                   |
+| array_has_any(array, sub-array)      | Returns true if any elements exist in both arrays `array_has_any([1,2,3], [1,4]) -> true`                                                          |
+| array_dims(array)                    | Returns an array of the array's dimensions. `array_dims([[1, 2, 3], [4, 5, 6]]) -> [2, 3]`                                                         |
+| array_fill(element, array)           | Returns an array filled with copies of the given value.                                                                                            |
+| array_length(array, dimension)       | Returns the length of the array dimension. `array_length([1, 2, 3, 4, 5]) -> 5`                                                                    |
+| array_ndims(array)                   | Returns the number of dimensions of the array. `array_ndims([[1, 2, 3], [4, 5, 6]]) -> 2`                                                          |
+| array_position(array, element)       | Searches for an element in the array, returns first occurrence. `array_position([1, 2, 2, 3, 4], 2) -> 2`                                          |
+| array_positions(array, element)      | Searches for an element in the array, returns all occurrences. `array_positions([1, 2, 2, 3, 4], 2) -> [2, 3]`                                     |
+| array_prepend(array, element)        | Prepends an element to the beginning of an array. `array_prepend(1, [2, 3, 4]) -> [1, 2, 3, 4]`                                                    |
+| array_remove(array, element)         | Removes one element from the array equal to the given value. `array_remove([1, 2, 2, 3, 1, 4], 2) -> [1, 2, 3, 1, 4]`                              |
```

Comment:
Maybe we could explicitly say "Removes the first element from the array equal to the given value":

```suggestion
| array_remove(array, element)         | Removes the first element from the array equal to the given value. `array_remove([1, 2, 2, 3, 1, 4], 2) -> [1, 2, 3, 1, 4]`                              |
```